src/webui/prefjson.cpp

```diff
@@ -114,6 +114,7 @@ QByteArray prefjson::getPreferences()
     data["ip_filter_enabled"] = session->isIPFilteringEnabled();
     data["ip_filter_path"] = Utils::Fs::toNativePath(session->IPFilterFile());
     data["ip_filter_trackers"] = session->isTrackerFilteringEnabled();
+    data["banned_ip_list"] = session->BannedIPs().join("\n");
```

Comment:
Is there such a Society? I would like to _enlist_ in it 😃.
I have put List everywhere because in my brain all processing I have coded is spinning around structured in List entities, not just scattered multiple entities. Briefly, I'm working with List here, not with IPs. 
Anyway I rename it as you wish.